### PR TITLE
VIH-8982 - added feedback to judge and jho waiting room pages

### PR DESCRIPTION
**Pull request by abbasiTahir**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8982

### Change description ###
Added the new feedback link for Judge and JHO on hearing waiting room page . 
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-space.module.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ import { ConfirmLeaveHearingPopupComponent } from './confirmation/confirm-leave-
 import { ParticipantRemoteMuteStoreService } from './services/participant-remote-mute-store.service';
 import { PrivateConsultationParticipantStatusComponent } from './participant-waiting-room/private-consultation-participants/private-consultation-participant-status/private-consultation-participant-status.component';
 import { PrivateConsultationParticipantDisplayNameComponent } from './participant-waiting-room/private-consultation-participants/private-consultation-participant-display-name/private-consultation-participant-display-name.component';
-import { FeedbackBannerComponent } from './participant-waiting-room/feedback-banner/feedback-banner.component';
+import { FeedbackBannerComponent } from './waiting-room-shared/feedback-banner/feedback-banner.component';
```
[the remark this answers is not in the document]
Comment (abbasiTahir) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-space.module.ts:39]: it's the refactoring of the common component .

File: VideoWeb/VideoWeb/ClientApp/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "version": "0.0.0",
     "scripts": {
         "ng": "ng",
-        "start": "ng serve -c development",
+        "start": "ng serve -c development --port 4400",
```
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/package.json:6]: Sorry this was a mistake.

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-space.module.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-space.module.ts:39]: we moved the feedback component to a shared folder to be used from other components

**Comment by will-craig** [comment]
Aside from removing the commits that update the app settings (not needed) - looks good to me! Refactoring the feedback component to be at higher shared level that the JOH/Judge waiting rooms can consume is definitely the best approach.

File: VideoWeb/VideoWeb/appsettings.Development.json [hunk 3]
```diff
@@ -15,6 +15,6 @@
   },
   "VhServices": {
     "EventHubPath": "eventhub-dev",
-    "NgBaseUri": "http://localhost:4200/"
+    "NgBaseUri": "http://localhost:4400/"
```
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/appsettings.Development.json:18]: sorry I missed it

**Review by will-craig — APPROVED** [review]

**Pull request merged**